Projects/b_u585i_iot02a_ntz/Src/ota_pal/ota_pal_stm32u5_ntz.c

```diff
@@ -1194,6 +1198,8 @@ OtaPalStatus_t otaPal_SetPlatformImageState( OtaFileContext_t * const pxFileCont
                         break;
 
                     default:
+                        uxOtaStatus = OTA_PAL_COMBINE_ERR( OtaPalSuccess, 0 );
```

Comment:
Should this be an error case rather than a success case?

Reply:
Agree. I'll update if I can pass OTA PAL test with fix.